<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalyzeWindow</class>
 <widget class="QDialog" name="AnalyzeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analyze Results</string>
  </property>
  <widget class="QPushButton" name="showGlobalFSC">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>66</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>Visualize</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
background: none;
border: none;
background-repeat: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/see.png</normaloff>:/icons/icons/see.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>67</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-image: url(&quot;/home/vilas/OFSC/icons/helpblue.png&quot;);
background: none;
border: none;
background-repeat: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="showOFSC">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>110</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>Visualize</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
background: none;
border: none;
background-repeat: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/see.png</normaloff>:/icons/icons/see.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-image: url(&quot;/home/vilas/OFSC/icons/helpblue.png&quot;);
background: none;
border: none;
background-repeat: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="showDirFSC">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>Visualize</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
background: none;
border: none;
background-repeat: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/see.png</normaloff>:/icons/icons/see.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-image: url(&quot;/home/vilas/OFSC/icons/helpblue.png&quot;);
background: none;
border: none;
background-repeat: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>199</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-image: url(&quot;/home/vilas/OFSC/icons/helpblue.png&quot;);
background: none;
border: none;
background-repeat: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="showDirectionalResolutionDistribution">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>Visualize</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
background: none;
border: none;
background-repeat: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/see.png</normaloff>:/icons/icons/see.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>60</y>
     <width>332</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignRight">
     <widget class="QLabel" name="label_2">
      <property name="cursor">
       <cursorShape>BusyCursor</cursorShape>
      </property>
      <property name="text">
       <string>Global  FSC</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QLabel" name="label">
      <property name="cursor">
       <cursorShape>WhatsThisCursor</cursorShape>
      </property>
      <property name="text">
       <string>Fourier Shell Occupancy</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QLabel" name="label_3">
      <property name="cursor">
       <cursorShape>WhatsThisCursor</cursorShape>
      </property>
      <property name="text">
       <string>Directional FSC  direction </string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QLabel" name="label_6">
      <property name="cursor">
       <cursorShape>WhatsThisCursor</cursorShape>
      </property>
      <property name="text">
       <string>Resolution distribution</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QLabel" name="label_7">
      <property name="cursor">
       <cursorShape>WhatsThisCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>WASHINGTON (Reuters) - A record 26 million Americans likely sought unemployment benefits over the last five weeks, confirming that all the jobs created during the longest employment boom in U.S. history were wiped out in about a month as the novel coronavirus savages the economy.

Thursday's weekly jobless claims report from the Labor Department will add to a growing pile of increasingly bleak economic data. It will come amid rising protests against nationwide lockdowns to control the spread of COVID-19, the potentially lethal respiratory illness caused by the virus.

President Donald Trump, who is seeking a second term in the White House in November's general election, has been anxious to restart the paralyzed economy. Trump on Wednesday applauded steps taken by a handful of Republican-led states to begin reopening their economies, despite warnings from health experts of a potential new surge in infections.

&quot;The U.S. economy is hemorrhaging jobs at a pace and scale never before recorded,&quot; said Scott Anderson, chief economist at Bank of the West in San Francisco. &quot;It compares to a natural disaster on a national scale.&quot;

Initial claims for state unemployment benefits probably totaled 4.2 million in the week ended April 18, according to a Reuters survey of economists. Still a figure that would have been seen as unimaginably high less than two months ago, it would be lower than the previous week's 5.245 million. Estimates in the survey for Thursday's data were as high as 5.50 million. </string>
      </property>
      <property name="text">
       <string>Directional filtered Map</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QLabel" name="label_8">
      <property name="cursor">
       <cursorShape>WhatsThisCursor</cursorShape>
      </property>
      <property name="whatsThis">
       <string>WASHINGTON (Reuters) - A record 26 million Americans likely sought unemployment benefits over the last five weeks, confirming that all the jobs created during the longest employment boom in U.S. history were wiped out in about a month as the novel coronavirus savages the economy.

Thursday's weekly jobless claims report from the Labor Department will add to a growing pile of increasingly bleak economic data. It will come amid rising protests against nationwide lockdowns to control the spread of COVID-19, the potentially lethal respiratory illness caused by the virus.

President Donald Trump, who is seeking a second term in the White House in November's general election, has been anxious to restart the paralyzed economy. Trump on Wednesday applauded steps taken by a handful of Republican-led states to begin reopening their economies, despite warnings from health experts of a potential new surge in infections.

&quot;The U.S. economy is hemorrhaging jobs at a pace and scale never before recorded,&quot; said Scott Anderson, chief economist at Bank of the West in San Francisco. &quot;It compares to a natural disaster on a national scale.&quot;

Initial claims for state unemployment benefits probably totaled 4.2 million in the week ended April 18, according to a Reuters survey of economists. Still a figure that would have been seen as unimaginably high less than two months ago, it would be lower than the previous week's 5.245 million. Estimates in the survey for Thursday's data were as high as 5.50 million. </string>
      </property>
      <property name="text">
       <string>3DFSC</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="Topframe">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(32, 74, 135);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="YaleButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;TagareLab webpage&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tagare Lab webpage&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/icons/icons/yale.png</normaloff>:/icons/icons/yale.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>33</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>16</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>Results Analysis</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>239</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-image: url(&quot;/home/vilas/OFSC/icons/helpblue.png&quot;);
background: none;
border: none;
background-repeat: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="showFilteredMap">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>Chimera</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
background: none;
border: none;
background-repeat: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/see.png</normaloff>:/icons/icons/see.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="show3DFSC">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>284</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="toolTip">
    <string>Chimera</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
background: none;
border: none;
background-repeat: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/see.png</normaloff>:/icons/icons/see.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>284</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-image: url(&quot;/home/vilas/OFSC/icons/helpblue.png&quot;);
background: none;
border: none;
background-repeat: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="selectDir">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
